src/main/java/mat/server/service/impl/MatContextServiceUtil.java

```diff
@@ -184,17 +185,32 @@ public boolean isCurrentCQLLibraryDraftable(CQLLibraryDAO cqlLibraryDAO,
         return isCurrentCQLLibraryEditable(cqlLibraryDAO, libraryId, false);
     }
 
+    public boolean isValidatable(Measure measure) {
+        return isFhirMeasureValidatabe(measure) || isQdmMeasureValidatable(measure);
+    }
+
+    private boolean isQdmMeasureValidatable(Measure measure) {
+        return !measure.isDraft() &&
+                measure.isQdmMeasure() &&
+                isVersionEligibleForFhirValidation(measure.getQdmVersion(), measure.getReleaseVersion()) &&
+                !BooleanUtils.isTrue(measure.getIsCompositeMeasure());
+    }
+
+    private static boolean isFhirMeasureValidatabe(Measure measure) {

```

Comment:
I don't follow why this method is static.

Reply:
yeah.. I moved code from another helper class which was static. will remove